products-service/lambda-functions/constants/sns.emails.ts

```diff
@@ -0,0 +1,2 @@
+export const SNS_EMAIL_MAIN = "ivan.lashuk2013@gmail.com";
```

Comment:
`dotenv` is used in import service, it can be used to read emails from .env file to eliminate hardcoding.